test/WidenImage.js

```diff
@@ -0,0 +1,78 @@
+import applib from '../build/applib'
+import assert from 'assert'
+import fixtureIO from './Utilities/FixtureIO'
+import styleMocking from './Utilities/StyleMocking'
+
+const maybeWidenImage = applib.WidenImage.maybeWidenImage
+let document = null
```

Comment:
I think this worked ok for me here: https://github.com/wikimedia/applib/pull/3/files#diff-c00bff523716354efebd88904a328d70R71